apps/examples/power/power_main.c

```diff
@@ -46,7 +48,41 @@
  ****************************************************************************/
 
 static int is_running;
+static volatile int cpu_stress_running;
 
+static int cpu_stress_thread(void *arg)
+{
+	int cpu_id = (int)arg;
+	cpu_set_t cpuset;
+	pthread_t this_thread = pthread_self();
+	char thread_name[32];
+	clock_t prev = clock();
+	clock_t now;
+
+#ifdef CONFIG_SMP
```

Comment:
Is this config checking necessary?
If we execute the codes below with the config disabled, what happen?